### PR TITLE
Corrections in large-scale documentation

### PR DESCRIPTION
**Pull request by kostasvar**
As the title says - the corrections are for the documentation of the functions of the large-scale suite.

### CODE REVIEW

File: docs/bbob-largescale/functions/source/index.rst [hunk 1]
```diff
@@ -515,15 +515,15 @@ __ bbobfunctiondoc_
...
+       - :math:`f_{16}(\mathbf{x}) = 10\left( \dfrac{1}{n} \sum_{i=1}^{n} \sum_{k=0}^{11} \dfrac{1}{2^k} \cos \left( 2\pi 3^k \left( z_i + 1/2\right) \right) - f_0\right)^3 +` \\ :math:`+\dfrac{10}{n}f_{pen}(\mathbf{x}) + \mathbf{f}_{\text{opt}}`
        - :math:`\mathbf{z} = \mathbf{R}\mathbf{\Lambda}^{1/100}\mathbf{Q}T_{\text{osz}}(\mathbf{R}(\mathbf{x} - \mathbf{x}^{\text{opt}}))` \\ :math:`\text{ \ \ \ \ \ \ with } \mathbf{R} = P_{11}B_1P_{12}, \mathbf{Q} = P_{21}B_2P_{22},`\\ :math:`f_0= \sum_{k=0}^{11} \dfrac{1}{2^k} \cos(\pi 3^k)`
 
     *  - Schaffers F7 Function
-       - :math:`f_{17}(\mathbf{x}) = \gamma(n) \times\left(\dfrac{1}{n-1} \sum_{i=1}^{n-1} \left(\sqrt{s_i} + \sqrt{s_i}\sin^2\left( 50 (s_i)^{1/5}\right)\right)\right)^2 +` \\ :math:`+ 10f_{pen}(\mathbf{x}) + \mathbf{f}_{\text{opt}}`
+       - :math:`f_{17}(\mathbf{x}) = \left(\dfrac{1}{n-1} \sum_{i=1}^{n-1} \left(\sqrt{s_i} + \sqrt{s_i}\sin^2\left( 50 (s_i)^{1/5}\right)\right)\right)^2 +` \\ :math:`+\dfrac{10}{n}f_{pen}(\mathbf{x}) + \mathbf{f}_{\text{opt}}`
```
Comment (nikohansen) [inline docs/bbob-largescale/functions/source/index.rst]: where does the `10/n` come from? I just checked in the original code and in the original documentation. They both have only `10` as a factor in front of `f_pen`.

Comment (kostasvar) [reply docs/bbob-largescale/functions/source/index.rst]: See [this](https://github.com/numbbo/coco/blob/49c633fdaef9a6a28ff70f67c409ff249ec0b2e6/code-experiments/src/f_schaffers.c#L197) line.

Comment (nikohansen) [reply docs/bbob-largescale/functions/source/index.rst]: Looks like a bug then, as the definitions should be the same as in the `bbob` test suite up to 40-D.

**Comment by brockho** [comment]
Just for completeness, the [corresponding line in the development branch](https://github.com/numbbo/coco/blob/development/code-experiments/src/f_schaffers.c#L113).

**Pull request merged**